### ISSUE TITLE
chrore: remove Deno.test abstractions 

### ISSUE THREAD

**Issue by erfanium** [denodrivers/deno_mongo#355]
we should directly use `Deno.test` function in our tests (not using helper functions like `testWithClient`).

because new deno-vscode testing feature can't detect these tests.

**Comment by lucsoft** [comment]
I would prefer using BDD

This would cleanup our tests more and 
Deno.Test can get noisy

**Comment by Jaysok** [comment]
bdd.ts seems nice.

Most js developers would be familiar with bdd style testing convention(using describe it and so on) which is supported in most of js testing frameworks like mocha, jest and jasmine.

And I don't know deno-vscode supports bdd style test code detection for now, but IMHO it seems almost certain that it will be supported in the future, since bdd is standard feature.

**Comment by lucsoft** [comment]
yeah i think there are some attempts at adding support for this lense in vscode
